crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -359,6 +355,41 @@ pub fn duplicate_script_directory(
     let mut file = File::create(manifest_path).expect("Failed to create file");
     file.write_all(modified_toml.as_bytes())
         .expect("Failed to write to file");
+}
+
+pub fn duplicate_script_directory(
```

Comment:
why do we need copy script directory function, and duplicate script directory function? they sound very redundant

Reply:
In `duplicate_workspace_directory`, I needed to reuse part of the code from `duplicate_script_directory`, but without creating a new temp dir, so that part was moved to `copy_script_directory`

Reply:
Let's then rename the functions to properly indicate what they do - as `duplicate_directory` and `copy_directory` on the face of it does mean exactly the same thing

Reply:
How about `copy_script_directory` and `copy_script_directory_to_tempdir`?

Reply:
makes sense